rss2.xml

```diff
@@ -5,6 +5,11 @@
   <channel>
     <title>{{ config.title }}</title>
     <link>{{ url }}</link>
+    <image>
+      <url>{{ icon }}</url>
+      <title>{{ config.title }}</title>
+      <link>{{ url }}</link>
+    </image>
```

Comment:
I just noticed atom.xml does check [existence of `icon`](https://github.com/hexojs/hexo-generator-feed/blob/d169992a27181e4ab3443bcaaf7b71496113f4c7/atom.xml#L4), so rss2 also need.

Reply:
I added part to check if the icon exists (`{% if icon %}`) in rss2.xml